src/main/kotlin/org/onflow/flow/sdk/models.kt

```diff
@@ -74,24 +74,22 @@ enum class SignatureAlgorithm(
 enum class HashAlgorithm(
     val algorithm: String,
     val outputSize: Int,
-    val id: String,
     val code: Int,
     val index: Int
 ) {
-    UNKNOWN("unknown", -1, "unknown", -1, 0),
-    SHA2_256("SHA-256", 256, "SHA256withECDSA", 1, 1),
-    SHA2_384("SHA-384", 384, "SHA384withECDSA", 1, 2),
-    SHA3_256("SHA3-256", 256, "SHA3-256withECDSA", 3, 3),
-    SHA3_384("SHA3-384", 384, "SHA3-384withECDSA", 3, 4);
+    UNKNOWN("unknown", -1, -1, 0),
+    SHA2_256("SHA-256", 256, 1, 1),
+    SHA3_256("SHA3-256", 256, 3, 2),
+    KMAC128("KMAC128", 128, 5, 3),
```

Comment:
The 128 in KMAC is actually the security level not the output size (I know it's confusing). Although KMAC has an arbitrary output size, we could set the output size here to 256, which is the minimum size for hash collisions.

```suggestion
    KMAC128("KMAC128", 256, 5, 3),
```

Reply:
I addressed this in https://github.com/onflow/flow-jvm-sdk/pull/69. Feel free to merge or cherry pick